index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Project 1: Set up and Mobile Version Skeleton</title>
+
+  <link rel="stylesheet" type="text/css" href="styles.css">
+
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="nav-list">
+        <li><a id="nav-bar-a" href="">Welcome</a></li>
+        <li><button type="button" id="nav-bar-btn"><img src="images/icon-menu.png" alt="A hamburguer icon" width="35" height="35"></button></li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="heading-section">
+    <h1 id="main-heading">Hey there. I'm Ulises <br><strong>I'm a Software Developer</strong></h1>
+
+    <p> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+
+    <ul id="icons-div">
+      <li><img src="images/twitter.png" alt="twitter icon"></li>
+      <li><img src="images/linkedin-icon.png" alt="linkedin icon"></li>
```

Comment:
- [ ] [OPTIONAL] ALSO, you should wrap the `<img>` tag inside an `<a>` tag

Reply:
Thanks!!